index_pricelist.html

```diff
@@ -0,0 +1,94 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title> base page </title>
+    <meta name="author" content="Anna Danielkiewicz">
+    <meta name="description" content="Beauty salon web page">
+    <link rel="stylesheet" href="style_qanda.css" type="text/css">    
```

Comment:
Have you created his `style_qanda.css`? The file is not in the pull request. Maybe you forgot to commit it?

This is what I see:

<img width="200" alt="no_css" src="https://user-images.githubusercontent.com/13404719/86213506-13c05f00-bb7a-11ea-9919-3889e9114afa.png">

This is fine if you are going to add css in a future PR. Just checking